examples/pm2-typescript/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "version": "1.0.0",
+  "private": true,
+  "dependencies": {
+    "nuxt-ts-edge": "latest",
+    "vue-property-decorator": "^7.3.0"
+  },
+  "scripts": {
+    "dev": "nuxt-ts",
+    "build": "nuxt-ts build",
+    "start": "nuxt-ts start",
+    "generate": "nuxt-ts generate",
+    "lint": "tslint --project ."
+  },
+  "devDependencies": {
+    "@types/node": "^10.12.18",
```

Comment:
The `@types/node` is shipped by `nuxt-ts` through `@nuxt/typescript`, so you can securely remove it here.